src/selectors/course.js

```diff
@@ -29,21 +30,20 @@ export const getFilteredCourses = createSelector(
   }
 );
 
-/*01.07.17 - Removed testing on filter, as courses should no longer be tagged*/
 export const getFilteredExternalCourses = createSelector(
-  [getCourseContext, getIconContext],
-  (courseContext, iconContext) => {
+  [getCourseContext, getIconContext, getFilter],
+  (courseContext, iconContext, filter = {}) => {
     return courseContext.keys().reduce((res, path) => {
       const coursePath = path.slice(0, path.indexOf('/', 2));
       const fm = courseContext(path).frontmatter;
-      if (fm.external != null) {
+      if (fm.external !== undefined) {
```

Comment:
Er det noen grunn til at du forandret fra `!= null` til `!== undefined`?

Reply:
Burde man ikke alltid bruke !== istedet for != i javascript? Samme som at man bør bruke === istedet for ==?

Reply:
Jo, du har egentlig rett, men mange har dette ene unntaket: Når man skal sjekke om en variable er null eller undefined, så kan man bruke `==` og  `!=`.
Se f.eks. http://adripofjavascript.com/blog/drips/equals-equals-null-in-javascript.html